props/blaster.h

```diff
@@ -164,8 +136,10 @@ class Blaster : public PROP_INHERIT_PREFIX PropBase {
   }
 
   virtual void Reload() {
-    shots_fired_ = 0;
     SaberBase::DoEffect(EFFECT_RELOAD, 0);
+    shots_fired_ = 0;
+    empty_ = false;
+    is_jammed_ = false;
```

Comment:
This is new, and I'm not sure if it's the way people want it or not.
Should reloading really unjam the gun too?